debezium-connector-oracle/pom.xml

```diff
@@ -46,6 +46,11 @@
             <groupId>org.antlr</groupId>
             <artifactId>antlr4-runtime</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.ehcache</groupId>
+            <artifactId>ehcache</artifactId>
+            <version>3.8.1</version>
```

Comment:
For now, this is fine so we can keep the build localized to just the Oracle connector; however once we near the end of this PR's lifecycle, we need to move the version to the main pom where we maintain the other versions.